multi-arch-builders/README.md

```diff
@@ -32,12 +30,42 @@ aws ec2 run-instances                     \
     --tag-specifications "ResourceType=instance,Tags=[{Key=Name,Value=${NAME}}]" \
     --block-device-mappings "VirtualName=/dev/xvda,DeviceName=/dev/xvda,Ebs={VolumeSize=${DISK}}" \
     > out.json
+```
+
+Wait for the instance to come up (`a1.metal` instances can take 5-10 minutes to
+come up) and log in:
+
+```bash
+INSTANCE=$(jq --raw-output .Instances[0].InstanceId out.json)
+IP=$(aws ec2 describe-instances --instance-ids $INSTANCE --output json \
+     | jq -r '.Reservations[0].Instances[0].PublicIpAddress')
+ssh "core@${IP}"
+```
+
+Make sure the instance came up fine and wait for the COSA image build
+to complete:
 
+```bash
+systemctl --failed
+sudo machinectl builder@
+journalctl --user -f # to watch image build
+podman images # to view built image
+```
+
+Now that the instance is up and COSA is built we can re-assign the
+floating IP address. This removes the IP from the existing instance
+(if there is one) so you'll want to make sure no jobs are currently
+running on the existing instance by checking to make sure Jenkins is
+idle (i.e. no multi-arch aarch64 jobs are running).
+
+```bash
 # Grab the instance ID and associate the IP address
 INSTANCE=$(jq --raw-output .Instances[0].InstanceId out.json)
+EIP='18.233.54.49'
+EIPID='eipalloc-4305254a'
 aws ec2 associate-address --instance-id $INSTANCE --allow-reassociation --allocation-id $EIPID
+```
 
+Now you should be able to `ssh "core@${EIP}"`.
 
-# ssh into the instance
-# NOTE: Just this once ignore the ssh host key changed warning if you see it.
-ssh core@18.233.54.49
+NOTE: Just this once ignore the ssh host key changed warning if you see it.
```

Comment:
Do we need instructions here to tear down the old builder (if there is one)?

Reply:
added another paragraph in the last commit